astro.config.mjs

```diff
@@ -421,6 +438,10 @@ export default defineConfig({
           directory: '/resources/'
         },
       },
+      {
```

Comment:
For some reason, Releases displays further to the left than all the other top level sections

Reply:
Ah, good catch. Yeah I intended to fix that so that links that are not within a directory will show up right. But I think that in the future, the Releases category will provide boilerplate and drop-in releases as individual topics, meaning that Releases does justify it's own category to be toggled open.